### ISSUE TITLE
Should the default value of aria-live be undefined instead of off?

### ISSUE THREAD

**Issue by joanmarie** [w3c/aria#2653]
A number of ARIA properties have a default/implicit value of "undefined" which means the property is not supported on that element. `aria-live` is not one of those properties, and its default value of "off" has this definition:

> Indicates that updates to the region should not be presented to the user unless the user is currently focused on that region.

That, to me, suggests that the element is a live region, it just shouldn't be automatically announced. 

Given that `aria-live` is a global property, and that most elements are not live regions, I think the default value of `aria-live` should be "undefined" rather than "off."

**Comment by rahimabdi** [comment]
@joanmarie that makes sense IMO. To your point, looking at the definition of attributes with a similar unsupported state that is distinct from a supported-but-not-enabled state:

- `aria-expanded`
    - undefined (default) The element does not own or control a grouping element that is expandable.
-  `aria-checked`
    - undefined (default) The element does not support being checked.
- `aria-pressed`
    - undefined (default) The element does not support being pressed.  

I think next steps would be:

- Modify `aria-live` IDL enumerated attribute reflection: the content attribute's keywords would remain "off"/"polite"/"assertive", however, the missing value default would now be null (Undefined state) rather than "off", and the invalid value default would remain "off"
- Update ARIA spec to document this additional value (which is really the absence of a value), and how it differs from "off". For example: "(undefined) The element is not a region that communicates changes or updates." 

Adding to future ARIA WG agenda to confirm the above approach and to discuss potential webcompat impact of implementations/platform APIs now needing to differentiate between null and "off" `aria-live` values.

**Comment by patrickhlauke** [comment]
This actually ties in with a conversation I had ages ago with @cookiecrook and others, after https://tetralogical.com/blog/2024/05/01/why-are-my-live-regions-not-working/ came out. The fact that the default value wasn't defined (and folks assume it's "off", but then also don't really understand what `aria-live="off"` *actually* means because it's such a misleading name) was one of the takeaways.

**Comment by spectranaut** [comment]
Discussed in yesterday's meeting: https://www.w3.org/2025/10/23-aria-minutes.html#2566
